pandas/tseries/tools.py

```diff
@@ -188,7 +188,7 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
 
     Parameters
     ----------
-    arg : string, datetime, array of strings (with possible NAs)
+    arg : list, tuple, 1-d array, or Series
```

Comment:
this also accepts the values that you took out (e.g. accepts scalar string/datetime)